skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -44,10 +40,36 @@ def pcoa(distance_matrix):
     species is present in two sites, that means that the sites are
     similar.).
 
+    Note that the returned eigenvectors are not normalized to unit length.
+
     Parameters
     ----------
     distance_matrix : DistanceMatrix
         A distance matrix.
+    method : str, optional
+        Eigendecomposition method to use in performing PCoA.
+        By default, uses SciPy's `eigh`, which computes exact
+        eigenvectors and eigenvalues for all dimensions. The alternate
+        method, `fsvd`, uses faster heuristic eigendecomposition but loses
+        accuracy. The magnitude of accuracy lost is dependent on dataset.
+    number_of_dimensions : int, optional
+        Dimensions to reduce the distance matrix to. This number determines
+        how many eigenvectors and eigenvalues will be returned.
+        By default, equal to the number of dimensions of the distance matrix,
+        as default eigendecompsition using SciPy's `eigh` method computes
+        all eigenvectors and eigenvalues. If using fast heuristic
+        eigendecomposition through `fsvd`, a desired dimension should be
+        specified. Note that the default eigendecomposition method `eigh`
+        does not natively support a specifying number of dimensions to reduce
+        a matrix to, so if this parameter is specified,
+        all eigenvectors and eigenvalues will be simply be computed with no
+        speed gain, and only the number specified by `number_of_dimensions`
+        will be returned. Specifying a value of `0`, the default, will
+        set `number_of_dimensions` equal to the number of dimensions of the
+        specified `distance_matrix`.
+    inplace : bool, optional
+        If true, centers a distance matrix in-place in a manner that reduces
+        computational and memory cost.
```

Comment:
This is a comment I had before, can you address it:

> As far as I can tell the inplace operations are done iteratively instead of using a vectorized operation, hence the benefits would only be limited memory not compute. I suggest changing this to "reduces memory consumption".